misc/sparse_matrix.py

```diff
@@ -0,0 +1,64 @@
+import taichi as ti
+
+ti.init(arch=ti.x64, print_preprocessed=True)
+
+n = 8
+
+K = ti.SparseMatrixBuilder(n, n, max_num_triplets=100)
+f = ti.SparseMatrixBuilder(n, 1, max_num_triplets=100)
+
+
+@ti.kernel
+def fill(A: ti.sparse_matrix_builder(), b: ti.sparse_matrix_builder(),
+         interval: ti.i32):
+    for i in range(n):
+        if i > 0:
+            A[i - 1, i] += -1.0
+            A[i, i] += 1
+        if i < n - 1:
+            A[i + 1, i] += -1.0
+            A[i, i] += 1.0
+
+        if i % interval == 0:
+            b[i, 0] += 1.0
+
+
+fill(K, f, 3)
+
+print(">>>>>>>> Before build: ")
+K.print_triplets()
+
+A = K.build()
+
+print(">>>>>>>> After build: ")
+print(A)
+
+print(">>>> Summation Test: ")
```

Comment:
```suggestion
print(">>>> Summation")
```
You need a uniform format for the hint strings in `print`.